### DISCUSSION TITLE
Will vibration come back to the emoji view?

### DISCUSSION THREAD

**Discussion by robotwombat** [florisboard/florisboard#1675] [Q&A]
In previous versions of Florisboard, before the revamp, there was vibration when tapping on stuff (emoji, the 'ABC' button, etc.) in the emoji panel.

When the panel was redone, the vibration was nevee added back. Will it make a return?

**Comment by patrickgold** [comment]
Yes it will, probably in the 0.3.15/16 bug fix releases. They would vibrate now too but I just forgot re-adding it (because emoji buttons inherit differently than normal keys).

**Reply by robotwombat** [reply]
Thank you for all of the hard work! I'm looking forward to that bugfix :)
Florisboard is a fantastic project

**Reply by robotwombat** [reply]
> Yes it will, probably in the 0.3.15/16 bug fix releases. They would vibrate now too but I just forgot re-adding it (because emoji buttons inherit differently than normal keys).

Hi Patrick! Is there any update on this?

**Reply by patrickgold** [reply]
See PR #1855, this PR addresses this issue and re-implements vibration for the emojis itself, the tab row and the delete/ABC keys. Will be released in 0.3.16-beta03 in a few days.

**Reply by robotwombat** [reply]
> See PR #1855, this PR addresses this issue and re-implements vibration for the emojis itself, the tab row and the delete/ABC keys. Will be released in 0.3.16-beta03 in a few days.

Thanks!!
